azure-pipelines.yml

```
@@ -11,6 +11,7 @@ trigger:
       - NEWS
       - UPGRADING
       - UPGRADING.INTERNALS
+      - README.md
```

```suggestion
      - *.md
```

and we can also create a rule to ignore the ones inside `ext/` asl well

---

✔